content/en/docs/3.features/2.deployment-targets.md

```diff
@@ -31,9 +31,9 @@ export default {
 We are also exposing `process.target` for module authors to add logic depending on the user target.
 ::
 
-## Server Hosting
+## Server Hosting (Using nodejs as a webserver backend)
 
-Server Hosting is hosting that requires a server and is intended for SSR applications or applications that are using [serverMiddleware](/docs/configuration-glossary/configuration-servermiddleware). Server-side rendering otherwise known as SSR means that your page is rendered on the server when it is requested by the user. When the user opens your page in a browser the browser sends a request to the server requesting that page. The page is rendered on the server and sent back to the browser with all its content.
+Server Hosting is hosting that requires a nodejs webserver and is intended for SSR applications or applications that are using [serverMiddleware](/docs/configuration-glossary/configuration-servermiddleware). Server-side rendering otherwise known as SSR means that your page is rendered through a nodejs webserver environment when it is requested by the user. When the user opens your page in a browser the browser sends a request to the nodejs webserver requesting that page. The page is rendered on the server and sent back to the browser with all its content.
```

Comment:
```suggestion
Server Hosting is hosting that requires a NodeJS webserver and is intended for SSR applications or applications that are using [serverMiddleware](/docs/configuration-glossary/configuration-servermiddleware). Server-side rendering otherwise known as SSR means that your page is rendered through a NodeJS webserver environment when it is requested by the user. When the user opens your page in a browser the browser sends a request to the nodejs webserver requesting that page. The page is rendered on the server and sent back to the browser with all its content.
```